mavlink-bindgen/src/parser.rs

```diff
@@ -636,6 +662,7 @@ impl MavEnum {
 
             impl #enum_name {
                 #const_default
+                #special_impl
```

Comment:
I think `special_impl` naming is not good here, it doesn't give the impression of `mav_bool`.